examples/quickstarts/nodejs-postgresql-persistent.json

```diff
@@ -383,7 +383,7 @@
 								},
 								"volumeMounts": [
 									{
-										"mountPath": "/var/lib/postgresql/data",
```

Comment:
Are the image expecting this path change?

Reply:
Not a change, but a bugfix; see https://github.com/nodeshift-starters/nodejs-rest-http-crud/issues/228.  Note that this brings it in line with other pg-based templates.